index.html

```diff
@@ -4,7 +4,194 @@
+        <a href="#"><img id="hamburguer-icon" src="./assets/images/lista.png" alt="menu" /></a>
+      </nav>
+    </header>
+
+    <div class="hamburguer">
+      <button id="btn-close" type="button" class="btn-close" aria-label="Close"></button>
+      <ul>
+        <li>
+          <a class="hamburguer-link" href="index.html"><h3 class="h3">Home</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#mainProgram"><h3 class="h3">Main Program</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#featuredPerformers"><h3 class="h3">Featured Performers</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="about.html"><h3 class="h3">About</h3></a>
+        </li>
+      </ul>
+    </div>
+
+    <main>
+      <section id="heading" class="heading">
+        <h2>"Hello Reggae Lover"</h2>
+        <h1>
+          Protoje Reggae <br />
+          Munich Concert <br />
+          2023
+        </h1>
+        <p class="heading">
+          Munich is a vibrant city with a rich cultural history, and reggae music has played a significant role in shaping its
+          music scene.
+        </p>
+        <h3>2022.03.27(FRI) and 28(SAT)</h3>
+        <h4>@Olympiahalle Arena, Munich - Deutschland</h4>
+      </section>
+
+      <section id="mainProgram" class="main_program">
+        <h2 class="main_program">Main Program</h2>
+        <hr class="hr1" />
+        <ul class="main_program">
+          <li class="main_program">
+            <div class="main_program">
+              <img src="./assets/images/writing.png" alt="Autograph Session" />
+            </div>
+            <h2>Autograph Session</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program">
+            <div class="main_program">
+              <img src="./assets/images/video-camera.png" alt="New Video Recording" />
+            </div>
+            <h2>Video Recording</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program">
+            <div class="main_program">
+              <img src="./assets/images/ear.png" alt="Test Sound" />
+            </div>
+            <h2>Test Sound</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program col-sm-12 col-md-6 col-lg-2">
+            <div class="main_program">
+              <img src="./assets/images/concert.png" alt="Concert" />
+            </div>
+            <h2>Concert</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program col-sm-12 col-md-6 col-lg-2">
+            <div class="main_program">
+              <img src="./assets/images/reggae.png" alt="After Party" />
```

Comment:
- [ ] As per the project requirements, we have to stick to Cindy's design as much as possible, therefore,I suggest using the background that matches most.
Hint: you can apply this one: ![download (1)](https://user-images.githubusercontent.com/97834160/211685809-8347379a-acf7-470c-89ed-b2bf9146f445.svg)


  - Your design:
![Ybg](https://user-images.githubusercontent.com/97834160/211685919-2ccc3e35-1ed6-4f45-a8a4-20a9250692c8.png)


  - Behance design:
![Bbg](https://user-images.githubusercontent.com/97834160/211685902-7e981311-1f6c-464b-aa93-26aea4b47996.png)

Reply:
Dear @gedeonTS unfurtunaly I don't have the original background because the template is not from Fidga. What can I do to get the same background.? I used the same color from the original background.